config/config-mp/src/main/java/io/helidon/config/mp/MpPropertiesMetaConfigProvider.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.config.mp;
+
+import java.util.List;
+import java.util.Set;
+
+import io.helidon.config.Config;
+import io.helidon.config.mp.spi.MpMetaConfigProvider;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+/**
+ * Helidon MicroProfile meta-config provider for Properties configuration files.
+ */
```

Comment:
Give this provider a low priority (and all other default providers) so it is easy to override by users. This can be done by implementing Prioritized.

Reply:
Is this about adding 
```
@Priority(300) 
```
in the class? Are the default providers include System Properties?

Reply:
yes, all the meta config providers we add explicitly should have low priority, so if anybody creates a custom provider with the same type, theirs would be first. 
I would implement `Prioritized` and returns the value (as you  are adding instances and not classes).

Reply:
The meta-config has a `ordinal` parameter which I think also helps in reordering? Is that different from this?

Reply:
The priority #s I added are:
1. MpEnvironmentVariablesMetaConfigProvider - 50
2. MpSystemPropertiesMetaConfigProvider - 51
3. MpPropertiesMetaConfigProvider - 52
These are the ones we add explicitly using HelidonServiceLoader. Do the nos. look ok? YamlMetaConfigProvider and HoconMpMetaConfigProvider are added via `provides...` in module-info.java, so I'm assuming we don't need to add to those?

Reply:
Please check if what I did to implement Prioritized is ok and if the low numbers used are acceptable.

Reply:
As discussed, also implemented `Prioritized` on `HoconMpMetaConfigProvider` and `YamlMetaConfigProvider` and set priority to 300 for all MetaConfigProviders